docs/src/main/docbook/client.xml

```diff
@@ -999,4 +999,139 @@ Client client = ClientBuilder.newBuilder().sslContext(sslContext).build();</prog
 
 
     </section>
+
+    <section>
+        <title>New Features</title>
```

Comment:
Please do not say it's a new feature. The documentation is there for good, it will be an old feature in a few releases.